specs/swagger.yml

```diff
@@ -0,0 +1,1397 @@
+---
+swagger: '2.0'
+info:
+  version: 0.16.54
+  title: Ziti Fabric
+  contact: {}
+host: demo.ziti.dev
+basePath: /fabric/v1
+schemes:
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+
+#######################################################################################################################
+#
+# Paths - Endpoints and their associated HTTP methods
+#
+#######################################################################################################################
+paths:
+  ###################################################################
+  # Services
+  ##################################################################
+  '/services':
+    get:
+      summary: List services
+      description: |
+        Retrieves a list of service resources; supports filtering, sorting, and pagination. Requires admin access.
+      tags:
```

Comment:
What kind of auth is required? Just a client cert?

Reply:
Yes, when in pure fabric mode only a valid client cert is required (like current ziti-fabric). When running with the edge it piggy backs on edge authentication and requires a valid api session. 

I was concerned about removing the security section, but from I can see in the edge code we just ignore that parameter and enforce the presence of a valid ztSession in our code, so I thought it was ok.

Reply:
Open API 2.0 / Swagger doesn't have a way to specify client cert auth. The only option you have is to document it on every endpoint with words.

Reply:
It is interesting that this API has two security modes though. I'm not sure how to document it w/o having two specs.